src/main/scala/com/workday/warp/dsl/WarpMatchers.scala

```diff
@@ -127,19 +131,20 @@ trait WarpMatchers {
     *
     * @param negated a [[ResultOfNotWordForAny]] representing a negated match condition.
     */
-  implicit class NegatedMatcher[T <: TrialResult[Any]](negated: ResultOfNotWordForAny[List[T]]) {
+  implicit class NegatedMatcher[T <: Iterable[TrialResult[_]]](negated: ResultOfNotWordForAny[T]) {
 
     /** Part of the dsl. Allows syntax like `should not exceed (64 seconds)`. Uses scalatest dsl to create a
       * [[Matcher]] and a [[MatchResult]] for the measured response time.
       *
       * @param threshold maximum response time threshold.
       */
     @DslApi
-    def exceed(threshold: Duration): Unit = {
-      this.negated be BeMatcher[List[TrialResult[_]]] { left: List[TrialResult[_]] =>
-        ResponseTimeCollector.updateThresholds(left, threshold)
+    def exceed(threshold: Duration)(implicit ev: ClassTag[T]): Assertion = {
 
-        val measuredResponseTimes: List[Option[Duration]] = left map { _.maybeResponseTime }
+      this.negated be BeMatcher[T] { ogLeft: T =>
```

Comment:
what's `ogLeft` represent?

Reply:
`ogLeft` represents the left side of a comparison. its kind of hard to read, but we are creating a `Matcher` and passing that to `be`.

`BeMatcher.apply[T]` takes a comparison function that represents the left side of the comparison:

```
object BeMatcher {
  def apply[T](fun: T => MatchResult)(implicit ev: ClassTag[T]): BeMatcher[T]
}
```

Reply:
could we call `ogLeft`, `source` or `base` instead? I googled `left side of a comparison function is called` and got this as one of the results: https://english.stackexchange.com/questions/392587/whats-the-term-for-the-left-hand-side-of-a-comparison-operation. `ogLeft` doesn't really tell me much.

Reply:
we can call it `left` since thats what scalatest calls it https://github.com/scalatest/scalatest/blob/3.1.x/scalatest/src/main/scala/org/scalatest/matchers/BeMatcher.scala

@sarcasticsimba can you rename 
`left` -> `trials`
`ogLeft` -> `left`
?